ui-react/packages/atlasmap-ui/src/mapper/MapperContextToolbar.tsx

```diff
@@ -41,7 +44,8 @@ export const MapperContextToolbar: FunctionComponent<
           >
           <FilePicker
             extensions={['adm', 'jar']}
-            onChange={(selectedFile: File) => processImportedFile( {selectedFile} )}
+            // onChange={(selectedFile: File) => processImportedFile( {selectedFile} )}
+            onChange={(selectedFile: File) => importAtlasFile(selectedFile)}
```

Comment:
`onChange={importAtlasFile}` would be preferable since lambdas can create performance issues.

Check [tslint jsx-no-lambda
](https://github.com/palantir/tslint-react)

> Creating new anonymous functions (with either the function syntax or ES2015 arrow syntax) inside the render call stack works against pure component rendering. When doing an equality check between two lambdas, React will always consider them unequal values and force the component to re-render more often than necessary.